controllers/chat/deleteMessage.js

```diff
@@ -18,27 +18,27 @@ const deleteMessage = async (req, res) => {
     await client.partialUpdateMessage(messageID, {
       set: {
         message_type: MESSAGE_TYPE.DELETED,
-        text: 'This message has been deleted'
+        text: 'This message has been deleted',
+        attachments: []
       }
     });
-    let destroy = await client.deleteMessage(messageID);
+
     // sent empty message to channel
     let baseMessage = {
       user_id: req.userId,
-      message_type: 'deleted',
+      message_type: 'notification-deleted',
+      text: 'This message has been deleted',
       deleted_message_id: messageID,
-      deleted_message_created_at: message?.created_at,
-      deleted_message_updated_at: message?.updated_at
+      deleted_message_created_at: message?.message?.created_at,
+      deleted_message_updated_at: message?.message?.updated_at
     };
+
+    console.log('baseMessage', baseMessage);
     const channel = client.channel('messaging', channelId);
     await channel.sendMessage(baseMessage, {
       skip_push: true
     });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendMessage` function is being called with a second argument `{ skip_push: true }`. However, in the old hunk, this function was called without this argument. Make sure this change is intentional and won't cause any unexpected behavior.

No error handling is done for the `sendMessage` function. If this function throws an error, it will propagate up and might cause the request to fail. Consider adding error handling here to improve the robustness of your code.



<!-- This is an auto-generated comment by OSS CodeRabbit -->